src/pages/PlayerNames.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import { motion } from "framer-motion";
+import "./PlayerNames.scss"; // Import the SCSS file
+import Game from "./Game"; // Import the Game component
+import close from "../img/close.png"; // Corrected import statement
+import o from "../img/o.png"; // Corrected import statement
+
+const PlayerNames = () => {
+  const [playerX, setPlayerX] = useState("");
+  const [playerO, setPlayerO] = useState("");
+  const [gameStarted, setGameStarted] = useState(false);
+
+  const handleInputChange = (e, setPlayer) => {
+    setPlayer(e.target.value);
+  };
+
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+
+    // Save player names to local storage
+    localStorage.setItem("playerX", playerX);
+    localStorage.setItem("playerO", playerO);
+
```

Comment:
no need to store names in the local storage, just pass the names to the next page